src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/Blocks/View/addresses.html.twig

```diff
@@ -0,0 +1,80 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="card">
```

Comment:
Why not using a Grid here?

Reply:
Actually we did not think about it 🤔 can be used indeed although it's a bit like using a RPG to kill a fly as Grid is very powerful with lot of features while this is a very basic view with no interactions

Reply:
ummmm, hard question. :smile: This grid was not hookable so it still isn't i guess.

Reply:
I know it' overkill in this case, although you can have a simple GridDefinition without filters or actions The main advantage is that it's easy to add/remove columns

Reply:
Even though it might be nice to have, but this is not something critical or very important, so let's focus on current tasks as we are already overwhelmed with other PRs to get them merged before freeze. :/